linux_os/guide/system/software/sudo/sudo_restrict_privilege_elevation_to_authorized/rule.yml

```
@@ -0,0 +1,40 @@
+documentation_complete: true
+
+title: 'The operating system must restrict privilege elevation to authorized personnel'
+
+prodtype: sle15
+
+description: |-
+    The sudo command allows a user to execute programs with elevated
+    (administrator) privileges. It prompts the user for their password
+    and confirms your request to execute a command by checking a file,
+    called sudoers.
+
+rationale: |-
+    If the "sudoers" file is not configured correctly, any user defined
+    on the system can initiate privileged actions on the target system.
+
+severity: medium
+
+identifiers:
+    cce@sle15: CCE-85712-8
+
+references:
+    nist: CM-6(b),CM-6(iv)
+    disa@sle15: CCI-000366
+    stig@sle15: SLES-15-020101
+
+ocil_clause: 'Verify the "sudoers" file restricts sudo access to authorized personnel.'
```

The `ocil_clause` parameters renders the question:

`Is it the case that <ocil_clause> ?`

As it can be seen in the full rendered OCIL question text below:

```      
      <ocil:question_text>To determine if "sudoers" file, restricts sudo access, run the following commands:
$ sudo grep -PR '^\s*ALL\s+ALL\=\(ALL\)\s+ALL\s*$' /etc/sudoers /etc/sudoers.d/*
$ sudo grep -PR '^\s*ALL\s+ALL\=\(ALL\:ALL\)\s+ALL\s*$' /etc/sudoers /etc/sudoers.d/*
Both commands should return no output.
      Is it the case that Verify the "sudoers" file restricts sudo access to authorized personnel.?
      </ocil:question_text>
```

```suggestion
ocil_clause: '/etc/sudoers file does not restrict sudo access to authorized personnel'
```